how-to.md

```diff
@@ -134,10 +134,10 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
 
 ## Revert your tenant to a point in time
 
-* :warning: **Warning**	:warning: this is a destructive operation that can result in deleting data (such as users) from your tenant(s). Please carefully review the pending changes on the Sync page before approving
-* Navigate to https://dev.azure.com/ > **Tenants** > **Pipelines** > select the tenant that you want to revert > **Run pipeline** > **master** > **Tags** > enter the date of the commit you want to revert to (e.g. 20210112.1) > select the commit you want to revert by clicking the date > **Run** > several minutes later you will be required to approve the Sync to make the changes to your tenant
+* :warning: **Warning**	:warning: this is a destructive operation that can result in deleting data (such as users) from your tenant(s).
```

Comment:
Is the format of these steps consistent with the other how to guides? Should be consistent with other how to.

The other how tos seem to either bullet out each step or use arrows, whereas you seem to be writing it out in sentence form

Reply:
I made some other comments directly on the code, but only submitted this as a review, so be sure to address those comments as well @brad-Simeon

Reply:
Thanks, I believe I have incorporated all comments, let me know if I am missing anything @david-simeon.